net-misc/curl/curl-7.88.1-r1.ebuild

```diff
@@ -93,6 +93,10 @@ MULTILIB_CHOST_TOOLS=(
 PATCHES=(
```

Comment:
I'd consider no `git mv` for this in case of unexpected regressions in these. But the patches are also tiny, so it's your call. Not insisting here.